wikihow2zim/scraper.py

```diff
@@ -542,6 +542,10 @@ def scrape_article(self, article, remove_all_links=False):
         for selector in black_list:
             _ = [elem.decompose() for elem in content.select(selector)]
 
+        # Remove link of page of author
+        for link in content.select("#byline_info > a[href]"):
```

Comment:
That ID in the selector designate any info in *byline*. We should try to be more specific:

`#coauthor_byline a[href]` seems more appropriate for instance